README.md

```diff
@@ -102,6 +102,7 @@ By default the enricher uses the following masking operators:
 - EmailAddressMaskingOperator
 - IbanMaskingOperator
 - CreditCardMaskingOperator
+- PathMaskingOperator
```

Comment:
Adding the default operators when I built this library was a bit of a mistake.
Could you remove it from the docs here? (I see you didn't actually add it to the list of default operators)

Reply:
Added to the default list of operators